app/assets/stylesheets/application.scss

```diff
@@ -1,46 +1,67 @@
 @import "bootstrap-sprockets";
 @import "bootstrap";
 
```

Comment:
I'm hoping there won't be much more than this, if it gets much larger than this I will break it up. But right now, I can't really see a smooth breaking point.